viadot/tasks/vid_club.py

```diff
@@ -100,6 +100,11 @@ def run(
             items_per_page (int, optional): Number of entries per page. 100 entries by default.
             region (str, optional): Region filter for the query. Valid inputs: ["bg", "hu", "hr", "pl", "ro", "si", "all"]. Defaults to "all".
             days_interval (int, optional): Days specified in date range per api call (test showed that 30-40 is optimal for performance). Defaults to 30.
+            cols_to_drop (List[str], optional): List of columns to drop. Defaults to None.
+
+        Raises:
+            ke: When DataFrame doesn't contain columns provided in the list of columns to drop.
```

Comment:
Please replace `ke` with `KeyError` in docstring.

Reply:
ok